src/demo-app/datepicker/datepicker-demo.ts

```diff
@@ -6,36 +6,58 @@
  * found in the LICENSE file at https://angular.io/license
  */
 
-import {ChangeDetectionStrategy, Component} from '@angular/core';
+import {ChangeDetectionStrategy, Component, Host} from '@angular/core';
 import {FormControl} from '@angular/forms';
 import {MatDatepickerInputEvent} from '@angular/material/datepicker';
+import {DateAdapter} from '@angular/material/core';
+import {MatCalendar} from '@angular/material';
 
 
 @Component({
-  moduleId: module.id,
-  selector: 'datepicker-demo',
-  templateUrl: 'datepicker-demo.html',
-  styleUrls: ['datepicker-demo.css'],
-  changeDetection: ChangeDetectionStrategy.OnPush,
+    moduleId: module.id,
```

Comment:
Your formatting should be 2 spaces instead of a tab or four characters.

Reply:
@Chan4077 Thanks for pointing that out. I changed the indentation to two spaces, undoing the changes introduced previously thereby.